src/extension/settingsHelper.ts

```diff
@@ -96,4 +97,20 @@ export class SettingsHelper {
 
         return "";
     }
+
+    /**
+     * We get the packager port configured by the user
+     */
+    public static getPackagerStatusIndicatorPattern(fsPath: string): string {
+        const projectRoot = SettingsHelper.getReactNativeProjectRoot(fsPath);
+        let uri = vscode.Uri.file(projectRoot);
+        const workspaceConfiguration = vscode.workspace.getConfiguration("react-native.packager", uri);
+        if (workspaceConfiguration.has("status-indicator")) {
+            const version = ConfigurationReader.readString(workspaceConfiguration.get("status-indicator"));
+            if (version == PackagerStatusIndicator.FULL_VERSION || version == PackagerStatusIndicator.SHORT_VERSION) {
```

Comment:
Are you sure that `==` compare will work good here? What if we try to pass different types(not string) with the same value?